tools/bcz.cmd

```diff
@@ -116,8 +116,12 @@ set "__PROJECT_NAME=!_OUTPUT!"
 rem If we're trying to clean build, make sure to update the target here.
 if "%_MSBUILD_TARGET%" == "Build" (
     set __MSBUILD_TARGET=%__PROJECT_NAME%
-) else if "%_MSBUILD_TARGET%" == "Clean,Build" (
+) else if "%_MSBUILD_TARGET%" == "Clean;Build" (
     set __MSBUILD_TARGET=%__PROJECT_NAME%:Rebuild
+) else (
+    echo.
+    echo Oops... build bug in the neighborhood of configuring a build target.

```

Comment:
```suggestion
    echo Could not find a build target. This shouldn't happen.
```

I'm only suggesting updated text because "Oops" won't pass the spellcheck lol

Reply:
No... "oops" should pass spell check. My first iteration included the word "ooops" (3 'o's), so I *very quickly* corrected it to be the normal, non-onomatopoetic spelling ("oops" with just 2 'o's). Note that the spellchecker made a second "misspellings found" comment... but complaining about the original spelling (from the first iteration), **not** the fixed spelling. It must be confused by my blazing speed.

Not sure how to kick it... I'll try force-pushing or something...